src/components/Navbar/Navbar.styled.ts

```diff
@@ -1,60 +1,92 @@
 import styled from "styled-components";
 
-import { typography } from "../../styles/typography";
 import { breakpoints, themes } from "../../themes";
 
-export const NavbarContainer = styled.nav`
-	background-color: transparent;
+export const NavContainer = styled.nav`
 	display: flex;
 	justify-content: space-between;
 	align-items: center;
-	width: 100%;
+	padding: 0 10px 0 10px;
+	background-color: transparent;
+	transition: background-color 0.3s ease;
+
+	@media (max-width: ${breakpoints.tablet}) {
+		padding: 0.5rem 1rem;
+	}
 `;
 
 export const Logo = styled.img`
-	width: 120px;
-	height: 100px;
+	width: 100px;
+	transition: transform 0.3s ease;
 
-	@media (max-width: 768px) {
-		width: 30%;
-		margin-right: 2em;
+	&:hover {
+		transform: scale(1.05);
 	}
 `;
 
-export const NavLinks = styled.div`
+export const NavItems = styled.div`
 	display: flex;
-	width: 25%;
 	align-items: center;
-	justify-content: center;
-	margin-right: 32px;
-	margin-left: 48px;
+	gap: 16px;
 
 	@media (max-width: ${breakpoints.tablet}) {
-		flex-direction: row;
-		width: 70%;
-		margin-right: 0;
+		display: none;
+		position: absolute;
+		top: 100%;
+		left: 0;
+		right: 0;
+		flex-direction: column;
+		background-color: ${themes.light.background};
+		padding: 16px;
+		gap: 16px;
+
+		&.active {
+			display: flex;
+		}
 	}
 `;
 
-export const Button = styled.button`
-	background-color: ${themes.light.background};
-	font-family: ${typography.fontFamily.secondary};
-	padding: 10px 20px;
-	width: 120px;
-	border-radius: 5%;
+export const NavItem = styled.a`
 	color: ${themes.light.text};
-	border-radius: 25px;
-	outline: none;
-	font-weight: 700;
-	text-align: center;
+	text-decoration: none;
+	font-family: "Poppins", sans-serif;
```

Comment:
To odosobniony przypadek? Jeżeli każdy link będzie Poppinsem ustaw to proszę w globalnych ustawienaich CSS